packages/workflow-compile/bin/_truffle_workflow_compile

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+// This file is a simple CLI for running workflow-compile
+// while saving the artifact files **and** the compilation results
+// Usage: _truffle_workflow_compile <output-file-for-compilation-results>
+
+const path = require("path");
+const fs = require("fs");
+const Contracts = require("../new");
+const TruffleConfig = require("@truffle/config");
```

Comment:
Conventionally, we import this as `Config` instead of `TruffleConfig`, although that might not be true everywhere

Reply:
FYI @gnidan  https://github.com/trufflesuite/truffle/tree/develop/packages/config#usage
https://github.com/trufflesuite/truffle/blob/develop/packages/config/src/index.ts#L16
https://github.com/trufflesuite/truffle/blob/develop/packages/config/src/configDefaults.ts#L4